_includes/release_data.html

```diff
@@ -10,12 +10,12 @@
 }}
 {%- capture releaseChangelog -%}
 <ul>
-    <li>[media.ccc.de] Support <a href="https://streaming.media.ccc.de">livestreams</a></li>
-    <li>[media.ccc.de] Add "live" kiosk</li>
-    <li>[media.ccc.de] Add "recent" kiosk</li>
+    <li>Bottom tabs and second row of buttons</li>
+    <li>[PeerTube] Sepia Search</li>
+    <li>[YouTube] Chapters</li>
 </ul>
 {%- endcapture -%}
 
 {%- capture releasePostUrl -%}
-{{ site.baseurl }}/blog/pinned/release/newpipe-0.20.8/
+{{ site.baseurl }}/blog/pinned/release/newpipe-0.20.9/
```

Comment:
That should be 
```suggestion
{{ site.baseurl }}/blog/pinned/release/newpipe-0.20.9-released/
```
because you named your blog post file differently.

Reply:
its not easy